app-shells/fish/fish-2.5.0.ebuild

```diff
@@ -20,8 +20,12 @@ RDEPEND="
 DEPEND="
 	${RDEPEND}
 	sys-devel/bc
-	sys-devel/gettext
+	nls? ( sys-devel/gettext )
 "
+src_prepare() {
+	eapply "${FILESDIR}/${P}-honor-linguas.patch"
+	default
+}
```

Comment:
fixed!
I used single line since it's a single patch and most ebuilds I've checked also do so.